main.go

```diff
@@ -35,8 +35,16 @@ func getEnv(name string) string {
 }
 
 func createAwsConfig() *keystore.AwsConfig {
+	retryOnCredentialsError, _ := strconv.ParseBool(os.Getenv("AWS_RETRY_ON_CREDENTIALS_ERROR"))
```

Comment:
Is there any time you wouldn't want to retry?

Reply:
Figured as I don't know how to test this, it's better to disable it by default for now :man_shrugging: